app/pages/form/[id]/[page].tsx

```diff
@@ -46,10 +46,15 @@ const FormPage = ({ preloadedQuery }: RelayProps<{}, PageQuery>) => {
     <Layout session={session} title="Connecting Communities BC">
       <FormDiv>
         {status === 'withdrawn' && (
-          <StyledAlert id="review-alert" size="small" variant={'warning'}>
+          <StyledAlert id="review-alert" size="small" variant="warning">
             You can no longer edit this application because it is withdrawn.
           </StyledAlert>
         )}
+        {status === 'submitted' && (
+          <StyledAlert id="review-alert" size="small" variant="info">
+            Edits to this application are automatically submitted.
```

Comment:
Asked for confirmation re. wording in https://github.com/bcgov/CONN-CCBC-portal/issues/381#issuecomment-1232061010

Reply:
```suggestion
            Edits are automatically saved and submitted.
```

got confirmation that this needs to be updated

Reply:
Thanks I just pushed an update.